Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.Events.cs

```diff
@@ -138,6 +138,9 @@ private void InkScrollViewer_ViewChanged(object sender, ScrollViewerViewChangedE
         {
             if (!e.IsIntermediate)
             {
+
```

Comment:
Extra blank line causing build error:

InfiniteCanvas\InfiniteCanvas.Events.cs(140,13): error SA1505: An opening brace must not be followed by a blank line. [D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls\Microsoft.Toolkit.Uwp.UI.Controls.csproj]

Reply:
Thanks, I removed the line